src/js/rx/containers/Active.jsx

```diff
@@ -65,7 +68,7 @@ class Active extends React.Component {
     ];
 
     return (
-      <div className="rx-view-toggle">View:&nbsp;
+      <div className="rx-view-toggle" ref="viewToggle">View:&nbsp;
```

Comment:
Although this is easier to work with, I've generally been trying to follow React's guidelines of using the callback version of refs.

I guess I don't mind either way, but thoughts on following their recommendation in case they deprecate the old way of using refs?

> If you are currently using `this.refs.myRefName` to access refs, we recommend using this pattern instead.

https://facebook.github.io/react/docs/refs-and-the-dom.html